src/styles/icons.css

```diff
@@ -1,17 +1,18 @@
 /* Several icons in this set provided by Font Awesome by Dave Gandy - http://fontawesome.io */
+/* Several icons in this set provided by Icomoon by http://keyamoon.com */
 
 @font-face {
-  font-family: 'icomoon';
+  font-family: 'info-fonts';
   font-weight: normal;
   font-style: normal;
-  src: url('./fonts/info-icons.eot'); /* IE9 */
-  src: url('./fonts/info-icons.woff') format('woff'), /* Modern Browsers */
+  src: url('./fonts/info-icons.eot') format('eot'), /* IE9 */
+       url('./fonts/info-icons.woff') format('woff'), /* Modern Browsers */
```

Comment:
@katjam This second `src` is on purpose for old IE compatibility. It won't understand the second source so it just uses the first one. If all the URLs are in a single list it won't understand the whole thing.

Reply:
Ah - I think it was auto cleaned. Will replace.